src/Datadog.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {
+  BatchSize,
+  DatadogProvider,
+  DatadogProviderConfiguration, DdLogs,
+  SdkVerbosity,
+  UploadFrequency,
+} from '@datadog/mobile-react-native'
+import Config from 'react-native-config';
+import { DdRumReactNavigationTracking } from '@datadog/mobile-react-navigation'
+
+// Enable Datadog only in production, change to true to test locally
+const enabled = !__DEV__
+
+const clientToken = Config.DATADOG_CLIENT_TOKEN;
+if (enabled && !clientToken) {
+  throw new Error('DATADOG_CLIENT_TOKEN is required');
+}
+
+const applicationId = Config.DATADOG_APPLICATION_ID;
+if (enabled && !applicationId) {
+  throw new Error('DATADOG_APPLICATION_ID is required');
+}
+
+const datadogConfig = new DatadogProviderConfiguration(
+  clientToken,
+  __DEV__ ? 'dev' : 'prod',
```

Comment:
I am not sure we want to send logs in "dev" build? what is the use case for "dev builds? beta app?

Reply:
That's only for debugging the datadog setup during development. By default, it will be disabled and has to be enabled at https://github.com/coopcycle/coopcycle-app/pull/1846/files#diff-418d449e0322d2e2e548fcb698159345175ca7c9b097ce5ba96a159617dd37d1R13